api/views/admin_auth.py

```diff
@@ -1,13 +1,20 @@
 from flask import Blueprint, jsonify, request, make_response
 from flask_jwt_extended import (create_access_token, create_refresh_token,
-                                jwt_required, jwt_refresh_token_required, get_jwt_identity)
+                                jwt_required, jwt_refresh_token_required, get_jwt_identity,
+                                get_raw_jwt)
 from api.validators.user import validate_user
-from api import mongo, flask_bcrypt, jwt
+from api import mongo, flask_bcrypt, jwt, blacklist
 
 admin_auth = Blueprint("admin_auth", __name__)  # initialize blueprint
 users = mongo.db.users
 
 
+@jwt.token_in_blacklist_loader
+def check_if_token_in_blacklist(decrypted_token):
+    jti = decrypted_token['jti']
```

Comment:
A little confused on what jti is

Reply:
Ah JTI stands for JWT (json web token) ID - it's a common naming convention!